manifests/n/NumeRe/NumeRe/1.1.4.2209/NumeRe.NumeRe.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.4.2209
+PackageLocale: en-US
+Publisher: Erik Hänel et al.
+PublisherUrl: https://en.numere.org/
+PackageName: NumeRe - Framework für Numerische Rechnungen
+License: GNU GPL v3
+ShortDescription: Solving and visualizing. Table based. Statistics and numerics. Optimized for science. Free of charge. GNU GPL v3.
+Description: 'NumeRe: Framework für Numerische Rechnungen is an application for Microsoft Windows® that can do more than the usual spreadsheets. It provides you with nonlinear fits of arbitrary functions as well as a ODE solver. It can display 1D and 2D data easily and publication-ready with a simple command. Fourier transforms are included as well as wavelet transforms. Data is managed in a table-based manner and automatically saved, so you can quickly resume after a restart. SIMPLE TASKS ARE SIMPLE - We never understood why you have to write as much code for simple things as for more complex tasks. Our mantra is therefore Keep simple things simple. SYNTAX AS INTUITIVE AS A LANGUAGE - The main goal is to be as intuitive as possible, which implies a syntax that is as simple and clear as possible. NumeRe does not try to be dynamically typed, but deliberately emphasizes that you understand what is happening as soon as you read the code. In addition, the advanced editor highlights different data structures in different colors, so the syntax may seem a bit "colorful and choppy" at first. But we can guarantee that you will appreciate it very soon. STRUCTURE BUILT IN - When writing procedures, the deeply-integrated namespaces help you to keep order right away, because each folder automatically corresponds to a namespace. This makes it possible to have several procedures with the same name that are still unique. The editor supports you by auto-completion, as well as by displaying tooltips that show documentation and signatures of functions, commands and procedures.'

```

Comment:
```suggestion
Description: |-
  NumeRe Framework für Numerische Rechnungen is an application for Microsoft Windows® that can do more than the usual spreadsheets.
  It provides you with nonlinear fits of arbitrary functions as well as a ODE solver.
  It can display 1D and 2D data easily and publication-ready with a simple command
  Fourier transforms are included as well as wavelet transforms.
  Data is managed in a table-based manner and automatically saved, so you can quickly resume after a restart.
  SIMPLE TASKS ARE SIMPLE - We never understood why you have to write as much code for simple things as for more complex tasks.
  Our mantra is therefore Keep simple things simple. SYNTAX AS INTUITIVE AS A LANGUAGE - The main goal is to be as intuitive as possible, which implies a syntax that is as simple and clear as possible.
  NumeRe does not try to be dynamically typed, but deliberately emphasizes that you understand what is happening as soon as you read the code.
  In addition, the advanced editor highlights different data structures in different colors, so the syntax may seem a bit "colorful and choppy" at first. But we can guarantee that you will appreciate it very soon.
  STRUCTURE BUILT IN - When writing procedures, the deeply-integrated namespaces help you to keep order right away, because each folder automatically corresponds to a namespace. This makes it possible to have several procedures with the same name that are still unique.
  The editor supports you by auto-completion, as well as by displaying tooltips that show documentation and signatures of functions, commands and procedures.
```

Reply:
Hi @Trenly, thanks for the feedback. I was examining my manifests for previous versions on this repo and they all merged the multiline description in a single line (e.g. during manifest version update). However, as this PR is already merged, I'll change that for future releases.